http/src/main/resources/reference.conf

```diff
@@ -70,18 +70,28 @@ dataproc {
 
   monitor {
     initialDelay = 30 seconds
-    pollingInterval = 15 seconds
-    pollCheckMaxAttempts = 120 # 15 seconds * 120 is 30 min
-    checkToolsInterval = 8 seconds
-    checkToolsMaxAttempts = 75 # 8 seconds * 75 = 10 min
-    # Defines timeouts for cluster status transitions. If a status is not listed there is no timeout.
+    # Defines polling for runtime status transitions. Used commonly for all statuses.
```

Comment:
Changed some of these fields to use `PollMonitor` and `InterruptablePollMonitor` case classes.